### ISSUE TITLE
[Terminal]: Keyboard user is not able to access 'About' pop-up of terminal app.

### ISSUE THREAD

**Issue by v-rpundir** [microsoft/terminal#11259]
### Windows Terminal version (or Windows build number)

1.10.2383.0

### Other Software

**Test Environment:**
OS: Windows 11 Version 21H2 (OS Build 22000.168)
Screen Reader: Narrator
Tool: Accessibility Insight for Windows

### Steps to reproduce

**Repro Steps:**
Open Windows Terminal.
Verify that whether user can access about button and its pop-up using keyboard or not.

**User Experience:**
Keyboard users are impacted here as they are completely blocked to access about dialog.

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/keyboard

**Attachments:**
[Keyboard user is not able to access 'About' pop-up of terminal app..zip](https://github.com/microsoft/terminal/files/7184362/Keyboard.user.is.not.able.to.access.About.pop-up.of.terminal.app.zip)

### Expected Behavior

Keyboard user should be able to access 'About' pop-up of terminal app as it is accessible to mouse users.

### Actual Behavior

Keyboard user is not able to access 'About' pop-up of terminal app which is only accessible to mouse user.

Observation: Screen Reader user can access the about in Scan mode.

**Comment by elsaco** [comment]
@v-rpundir  there is no "About" option in the command palette also. Not sure if you referred to "shift+?" key cord only.

**Comment by elsaco** [comment]
@v-rpundir please try "crtl+shift+space" to open the drop-down menu and scroll to "About" when using the keyboard only. It is defined in default.json file:

`        { "command": "openNewTabDropdown", "keys": "ctrl+shift+space" },`

An option could be added to the command palette for "About" action. Is not there as of WT version 1.11.2421.0.

**Comment by v-rpundir** [comment]
Closing as agreeing as per above resolution.
